DRAFT_RELEASE_NOTES.md

```diff
@@ -50,6 +50,7 @@ To reduce burden of upgrading aiSSEMBLE, the Baton project is used to automate t
 | upgrade-v2-chart-files-aissemble-version-migration | Updates the helm chart dependencies within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE |
 | upgrade-v1-chart-files-aissemble-version-migration | Updates the docker image tags within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE       |
 | ml-flow-dockerfile-migration                       | Updates the MLFlow's Dockerfile to use the bitnami/mlflow image as a base instead of the deprecated boozallen/aissemble-mlflow image"                                      |
+| airflow-dockerfile-migration                       | Updates the Airflow's Dockerfile to use the bitnami/airflow image as a base instead of the deprecated boozallen/aissemble-airflow image"                                   |
```

Comment:
I forgot to add the airflow dockerfile migration to the readme as part of GH issue #227. Including it in this PR.

Reply:
Do you mind updating the milestone for the airflow ticket to be 1.9 so that it's clear  at a glance that it went in during 1.9 development.

Reply:
Just updated the milestone for #227 to 1.9.0.